README.md

```diff
@@ -375,6 +375,9 @@ This is how we create and publish a new release of the 121-platform.
   - ⚠️ **Note:**  
     Start with deployment of the "**_Demo_**"-instance.  
     This will **_also_** deploy the Mock-Service to its production-environment.
+- Send the "Inform stakeholders"-message to Teams in the following locations:
+  - Team: "510 CVA", channel: "General"
+  - Team: "NRK Thema Voedsel - 510 Support", channel: "General"
```

Comment:
```suggestion
- Send the "Inform stakeholders"-message to Teams in the locations mentioned in the DevOps item
```

I think we prefer not to talk about internal comm channels in the public documentation.